app/models/calculation_result.rb

```diff
@@ -67,9 +67,14 @@ def capital_upper_threshold
   end
 
   def client_assessed_capital
-    monetise(capital_row_items(prefix: "").values.compact.sum(0))
+    monetise([api_response.dig(:result_summary, :disposable_income, :assessed_capital), 0].compact.max)
   end
 
+  # Why is this being used instead of the assessed capital from the result? This looks like CCQ doing calculations??
+  # def client_assessed_capital
+  #   monetise(capital_row_items(prefix: "").values.compact.sum(0))
+  # end
+
```

Comment:
remove commented-out code. It was adding a+b+c+d because the total asset value wasn't available from CFE